src/components/ui/CustomSelect/SliderSelect.jsx

```diff
@@ -197,8 +197,13 @@ export default class CustomSelect extends React.Component {
         <span className="custom-select-text" onClick={this.toggle}>
           <div>
             <span>{this.state.selectedItem ? this.state.selectedItem.label : this.props.placeholder}</span>
+            {!this.state.selectedItem && this.state.closed &&
+              <svg className="c-icon -tiny icon-arrow-down" onMouseDown={this.toggle}>
```

Comment:
You have an `Icon` component, let's use it. And remove onMouseDown from it. Let's use a button that contains an Icon. What do you think @bitTal?